### ISSUE TITLE
Error?

### ISSUE THREAD

**Issue by Chan255** [thelinuxchoice/inshackle#17]
Anyone know a solve to this? Thanks.![Uploading 80271913...]()

**Comment by Hita6** [comment]
some times it work some times it doesn't i really dont why ... but when you remove the firt is of the celebgram_id just the first work ...

**Comment by Chan255** [comment]
Yeah every time I would try and remove the first celeb id it would just go
to the next

**Comment by jd114** [comment]
to solve this error go to inshackle folder celeb_id open delete all data and copy past below data

407964088
325734299
14455831
232192182
6590609
407964088
6860189
242598499
297604134
25025320
13460080

save it and try again If again the error accrues dm me

**Comment by jilalitajdine** [comment]
> to solve this error go to inshackle folder celeb_id open delete all data and copy past below data
> 
> 407964088
> 325734299
> 14455831
> 232192182
> 6590609
> 407964088
> 6860189
> 242598499
> 297604134
> 25025320
> 13460080
> 
> save it and try again...

Thanks it works fine now. appreciate
